meeting-management-agent/backend/services/google_calendar_service.py

```
@@ -0,0 +1,689 @@
+
+            # Validate that we got the essential tokens
+            if not token_info.get("access_token"):
+                raise ValueError("No access token received from Google")
+
+            logger.info("Successfully exchanged authorization code for tokens")
+            logger.info(f"Access token: {token_info['access_token'][:20]}...")
+            logger.info(
+                f"Refresh token: {'Present' if token_info.get('refresh_token') else 'Not present'}"
+            )
+
+            return token_info
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)